t/03-testapi.t

```diff
@@ -207,8 +208,8 @@ subtest 'record_info' => sub {
 subtest 'validate_script_output' => sub {
     my $mock_testapi = new Test::MockModule('testapi');
     $mock_testapi->mock(script_output => sub ($;$) { return 'output'; });
-    pass(validate_script_output('script', sub { m/output/ }));
-    pass(validate_script_output('script', sub { m/output/ }, 30));
+    is(validate_script_output('script', sub { m/output/ }), '', 'validating output with default timeout');
```

Comment:
possible to replace it with `ok(!...)` instead of `is(..,'')` 😃

Reply:
done